internal/api/plugins.go

```diff
@@ -33,11 +33,17 @@ func parsePluginFilter(u *url.URL) (*model.PluginFilter, error) {
 	filter := u.Query().Get("filter")
 	serverVersion := u.Query().Get("server_version")
 
+	enterprisePlugins, err := parseBool(u, "enterprise_plugins", false)
```

Comment:
I wonder if we can shorten this to just `enterprise`. But then maybe it becomes ambiguous between do i include enterprise plugins or do i only return enterprise plugins. What about `include_enterprise=true|false`?

Reply:
`enterprise_plugins` was defined and added in https://github.com/mattermost/mattermost-server/pull/14586, which shipped with `v5.24.0`. I'm 3/5 that `model.PluginFilter` should be a "copy" of http request. I could see value in changing the field in `GetPluginsRequest` to something like `include_enterprise`. But consistency between `GetPluginsRequest` and  `model.PluginFilter` seams more important to me. 1/5

Reply:
Ahh didn't know that webapp was already using this field. Agree that it's better to be consistent in that regard.